### DISCUSSION TITLE
How to rename subtitles Track#1 ,Track#2

### DISCUSSION THREAD

**Discussion by xiu2hd** [mifi/lossless-cut#1295] [Q&A]
hello sir,  
I have 2 subtiles file have file 1.english 2.arbic .I add those two subtitles in my video it's show Tarck#1 and Track#2 how can i rename those track to English1 and Arabic2

**Comment by mifi** [comment]
have you tried the "edit track metadata" function? Then change "language"?

**Reply by xiu2hd** [reply]
yes sir 
i try metadata key,tags but it's not working ,what to write in key for Eng and Arabic  
![file_22](https://user-images.githubusercontent.com/111622038/188363772-df05573a-0552-4c82-a6a9-8956d5a5086f.jpg)
![file_23](https://user-images.githubusercontent.com/111622038/188363774-ecd71e89-a79b-4965-8691-366eeaf02fca.jpg)
sir check those two  file that one i edit its show track1 and track2  how can i add like other pictur arbie english and many other language

**Reply by mifi** [reply]
I think the key should be `language` and value `English` or `Arabic`
